src/Report/EnabledMembers.php

```diff
@@ -0,0 +1,222 @@
+            $filteredList = $filteredList->filter('methodCount', $countFilter);
+        }
+        $this->extend('updateSourceRecords', $filteredList);
+        return $filteredList;
+    }
+
+    /**
+     * List the columns configured to display in the resulting reports GridField
+     *
+     * @return array
+     */
+    public function columns(): array
+    {
+        $columns = singleton(Member::class)->summaryFields() + [
+            'methodCount' => _t(__CLASS__ . '.COLUMN_METHOD_COUNT', '№ methods'),
+            'registeredMethods' => _t(__CLASS__ . '.COLUMN_METHODS_REGISTERED', 'Method names'),
+            'defaultMethodName' => _t(__CLASS__ . '.COLUMN_METHOD_DEFAULT', 'Default method'),
+            'skippedRegistration' => _t(__CLASS__ . '.COLUMN_SKIPPED_REGISTRATION', 'Skipped registration'),
+        ];
+        $this->extend('updateColumns', $columns);
+        return $columns;
+    }
+
+    /**
+     * Provides the fields used to gather input to filter the report
+     *
+     * @return FieldList
+     */
+    public function parameterFields(): FieldList
+    {
+        $methods = $this->getMethodClassToTitleMapping();
+        unset($methods[$this->getBackupMethodClass()]);
+        $parameterFields = FieldList::create(
+            TextField::create(
+                'Member',
+                singleton(Member::class)->i18n_singular_name()
+            )->setDescription(_t(
+                __CLASS__ . '.FILTER_MEMBER_DESCRIPTION',
+                'Firstname, Surname, Email partial match search'
+            )),
+            NumericField::create(
+                'Count',
+                _t(__CLASS__ . '.COLUMN_METHOD_COUNT', 'Number of registered methods')
+            )->setHTML5(true),
+            DropdownField::create(
+                'Methods',
+                _t(__CLASS__ . '.COLUMN_METHODS_REGISTERED', 'Registered methods'),
+                $methods,
+                ''
+            )->setHasEmptyDefault(true),
+            CheckboxField::create(
+                'Skipped',
+                _t(__CLASS__ . '.COLUMN_SKIPPED_REGISTRATION', 'Skipped registration')
+            )
+        );
+        $this->extend('updateParameterFields', $parameterFields);
+        return $parameterFields;
+    }
+
+    /**
+     * Create an array mapping authentication method Class Names to Readable Names
+     *
+     * @return array
+     */
+    protected function getMethodClassToTitleMapping(): array
+    {
+        $methods = singleton(MethodRegistry::class)->getMethods();
+        $methodsClasses = array_map(
+            function (MethodInterface $method): string {
+                return get_class($method);
+            },
+            $methods
+        );
+        $methodNames = array_map(
+            function (MethodInterface $method): string {
+                return $method->getName();
+            },
+            $methods
+        );
+        return array_combine($methodsClasses, $methodNames);
```

Comment:
Down to super minor comments. I think this could be nicer (and avoid looping twice) to just create an array in a loop:

```php
$output = [];

foreach (MethodRegistry::singleton()->getMethods() as $method) {
    $output[get_class($method)] = $method->getName();
}

return $output;
```